frappe/printing/doctype/print_format/patches/sets_wkhtmltopdf_as_default_for_pdf_generator_field.py

```diff
@@ -0,0 +1,7 @@
+import frappe
+
+
+def execute():
+	"""sets "wkhtmltopdf" as default for pdf_generator field"""
+	for pf in frappe.get_all("Print Format", pluck="name"):
+		frappe.db.set_value("Print Format", pf, "pdf_generator", "wkhtmltopdf")
```

Comment:
Is this required? Just use fallback value in the code instead?

Reply:
yes, fallback is already available in the code. but, if it is not set then when user opens print format if shows the field empty, so for better clarity/ux. I patched old formats. 

also, someone in future might check the pdf_generator field with assumptions that it will be set.

Reply:
Okay, at least set "update_modified=False"

Reply:
done